### PR TITLE
Initial deployment for Storage Analytics Solution

### PR DESCRIPTION
**Pull request by Volkanco**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Volkanco__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by krnese** [comment]
You need to add the azuredeploy.parameters.json file, and the metadata.json to the folder. These are missing.

**Comment by Volkanco** [comment]
added parameters.json and metadata to the branch

File: oms-azurestorageanalytics-solution/azureStorageAnalytics.json [hunk 1]
```diff
@@ -0,0 +1,2497 @@
...
+                          "name": "Normal",
+                          "threshold": "Default",
+                          "color": "#009e49",
+                          "isDefault": true
+                        },
+                        {
+                          "name": "Warning",
+                          "threshold": "60",
+                          "color": "#fcd116",
+                          "
```
Comment (krnese) [inline oms-azurestorageanalytics-solution/azureStorageAnalytics.json]: Remove this comment

Comment (Volkanco) [inline oms-azurestorageanalytics-solution/azureStorageAnalytics.json]: Why  should we  remove this  one, Its  defines the indicators next to lists in OMS designer?
.by the way I have removed azurestorageanalytics.json and upload is as azuredeploy.json as  azurequickstart guide.

File: oms-azurestorageanalytics-solution/azuredeploy.parameters.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
     "omsLogAnalyticsRegion": {
       "value": "eastus"
     },
+    "omsLogAnalyticsSku": {
```
Comment (krnese) [inline oms-azurestorageanalytics-solution/azuredeploy.parameters.json]: using "free" sku as the default value is causing the deployment to fail in this subscription. Please use one of the other (paid) sku's

**Comment by Volkanco** [comment]
changed SKu to standalone

**Comment by singhkays** [comment]
oms-azure-storage-analytics-solution would be a better name for the folder @Volkanco Can you update?

**Pull request merged**